frontend/src/components/auth/Login.tsx

```diff
@@ -89,77 +132,159 @@ const Login = () => {
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>{title}</h1>
-      <form>
+    <Flex>
+      <Box bg="gray.100" w="40vw" h="100vh">
```

Comment:
nit: unfortunately we dont have this documented but: perfer to use long form prop names as it's more clear. ie use `background` instead of `bg` and use `width` instead of `w`